src/test/resources/elasticsearchdata/create-template.json

```diff
@@ -39,6 +38,7 @@
           "type": "synonym",
           "synonyms": [
```

Comment:
Doesn't the ticket say they wanted to remove these synonyms too e.g. for street and close?